microsite/src/pages/plugins/index.tsx

```diff
@@ -29,50 +29,135 @@ const plugins = pluginsContext.keys().reduce(
 
 plugins.corePlugins.sort((a, b) => a.order - b.order);
 plugins.otherPlugins.sort((a, b) => a.order - b.order);
-//#endregion
-
-const Plugins = () => (
-  <Layout>
-    <div
-      className={clsx('container', 'padding--lg', pluginsStyles.pluginsPage)}
-    >
-      <div className="marketplaceBanner">
-        <div className="marketplaceContent">
-          <h2>Plugin Marketplace</h2>
-
-          <p>
-            Open source plugins that you can add to your Backstage deployment.
-            Learn how to build a <Link to="/docs/plugins">plugin</Link>.
-          </p>
+
+const Plugins = () => {
+  const allCategories: ChipCategory[] = [];
+  plugins.corePlugins.concat(plugins.otherPlugins).forEach(pluginData => {
+    const index = allCategories.findIndex(
```

Comment:
I think we can do this a different way, using sets for uniqnes instead.

```ts
const allCategoriesSet = new Set([...plugins.corePlugins, ...plugins.otherPlugins].map(({ category }) => category))
```

Reply:
That is way much bettter 😄 thanks